api/process.go

```diff
@@ -58,9 +50,18 @@ func (s *service) Start(ctx context.Context, args *task.StartArgs) (*task.StartR
 		}
 		state.JID = args.JID
 	}
+
+	pid, err := s.run(ctx, args)
+	state.PID = pid
+	if err != nil {
+		// TODO BS: this should be at market level
+		s.logger.Error().Err(err).Msgf("failed to run task, attempt %d", 1)
```

Comment:
there's no doom loop here anymore, so both the comment (about the doom loop) and attempt counter can go